demo/popcha-client/web/sonar-project.properties

```diff
@@ -0,0 +1,19 @@
+sonar.organization=dedis
+sonar.projectKey=dedis_popstellar_popchawebclient
+
+sonar.python.version=3
```

Comment:
You might want to put this along with other python-specific params, to have a clean config file :)